### PR TITLE
Thermostat real

### PR DESCRIPTION
**Pull request by macnow**

### CODE REVIEW

File: accessories/thermostat.js [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+
+HomeAssistantThermostat.prototype = {
+	
+	// Required
+	getCurrentHeatingCoolingState: function(callback) {
+
+    this.client.fetchState(this.entity_id, function(data){
+      if (data) {
+
+				currentHeatingCoolingState = Characteristic.CurrentHeatingCoolingState.OFF;
```
Comment (maddox) [inline accessories/thermostat.js]: Can you bring these indentions in to match everything else?

File: index.js [hunk 2]
```diff
@@ -136,6 +138,8 @@ HomeAssistantPlatform.prototype = {
           accessory = new HomeAssistantSwitch(that.log, entity, that, 'scene')
         }else if (entity_type == 'media_player' && entity.attributes && entity.attributes.supported_media_commands){
           accessory = new HomeAssistantMediaPlayer(that.log, entity, that)
+        }else if (entity_type == 'thermostat'){
```
Comment (maddox) [inline index.js]: Isn't this climate now?

Please be sure you're on the most recent version of Home Assistant while developing this.

Comment (technicalpickles) [inline index.js]: Yeah, it's climate now. thermostat is still around and deprecated, so maybe include both?

**Comment by robbiet480** [comment]
Closing this as it's a stale PR and `thermostat` is no longer supported by Home Assistant. Please feel free to open a new PR with similar changes made for the new `climate` component.

**Pull request closed without merging**